load_env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Quickly perform a smoke test on target environment by running each
```

Comment:
This must be updated e.g.: "Performs the whole suite of load tests on target environment..."